.github/workflows/stackaid.yml

```diff
@@ -0,0 +1,11 @@
+name: 'fund-on-stackaid'
+on:
+  push:
+    branches:
+      - main
```

Comment:
maybe update temporarily to trigger on this branch so we can see & check the recognized dependencies here?